.drone.yml

```
@@ -4,22 +4,22 @@ workspace:
 
 services:
   web:
-    image: fpfis/httpd-php-ci:${PHP_VERSION}
+    image: registry.fpfis.eu/fpfis/httpd-php:${PHP_VERSION}
```

The PHP_VERSION variable shouldn't be touched, we should do like in here:
https://github.com/openeuropa/multivalue_form_element/pull/1/files#diff-b54b39f1afced2465e1f3641db9d5bbf4f3a7fcf890996dfedd3c197bcb7f8c7R7

```suggestion
    image: registry.fpfis.eu/fpfis/httpd-php:${PHP_VERSION}-ci
```